analysers/trials/numberOfConstructInstances.py

```diff
@@ -0,0 +1,65 @@
+import sys
+import json
+import io
+import gzip
+import uuid
+
+from datetime import timedelta
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark import SparkConf
+
+def createQuery(sc, cassandraKeyspace, srcTable, experimentID, trialID):
+    queries = []
+    
+    dataRDD = sc.cassandraTable(cassandraKeyspace, srcTable) \
```

Comment:
You definitely need `Alluxio` because you apply different computations in different Script over the same data.